conda/resolve.py

```diff
@@ -624,20 +624,22 @@ def generate_update_count(self, C, specs):
 
     def generate_feature_metric(self, C):
         eq = {}  # a C.minimize() objective: Dict[varname, coeff]
-        total = 0
+        # Given a pair (dist, feature), assign a "1" score IF:
+        # - The dist is installed
+        # - The dist does NOT require the feature
+        # - At least one package in the group DOES require the feature
+        # - A package that tracks the feature is installed
```

Comment:
Thanks for including this comment! That's very helpful to countercheck the code against.